src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+
+    <hr />
+
+    {Object.keys(post.comments).length !== 0 ? (
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable